go/vt/schemamanager/tablet_executor.go

```diff
@@ -297,6 +331,29 @@ func (exec *TabletExecutor) executeOneTablet(
 	}
 }
 
+// executeOnlineSchemaChangeOneTablet will request the tablet to run an online schema change
+func (exec *TabletExecutor) executeOnlineSchemaChangeOneTablet(
```

Comment:
This function can be changed to invoke `VExec` to insert a row in the `ghost` table (as mentioned in the issue). We can do a pairing session for doing the vttablet part that picks up this change and launches ghost. This approach will allow the schema migration to survive failovers.

Reply:
:+1: I need to learn more about that